test/test-build.js

```diff
@@ -39,6 +39,18 @@ function doTests(transpiler) {
     });
   });
 
+  test('Static string build', function () {
+    builder.reset();
+    builder.config({ transpiler: transpiler });
+    return builder.bundle('foo.js', {
+      fetch: function (data) {
+        if (data.name.indexOf('foo.js') !== -1) {
+          return fs.readFileSync('test/fixtures/test-tree/amd-1.js', 'utf8');
+        }
+      }
```

Comment:
My only thought was being able to provide a complex expression or a glob, then provide the source as a string to a select few files. I could not be entirely understanding the system though.

I will look into the suggestion a bit more today and try to update this PR :smile: